templates/ministerial/ministerial-en.html

```diff
@@ -33,113 +28,26 @@ <h2>Contact information</h2>
 				<strong>Email:</strong> <a href="mailto:">[first.last@canada.ca]</a> <span class="glyphicon glyphicon-envelope"></span></p>
 		</section>
 	</div>
-	<div class="col-md-3">
-		<h2>Twitter</h2>
-		[Twitter feed]
-	</div>
 </div>
 <div class="row">
```

Comment:
Please remove this wrapping div (and don't forget its closing tag), it makes the content overflow on both sides.
```suggestion

```